tests/test_cli.py

```diff
@@ -96,6 +96,35 @@ def test_cli_verbose(cli_runner):
     assert 'Project name: **Fake Project**' in content
 
 
+def test_user_config_recurse_submodules(mocker, cli_runner, user_config_path):
```

Comment:
What this test check? That you don't crash when use `--recurse-submodules`?

Reply:
No, it checks the arguments sent to the cookiecutter function when we use the `--recurse-submodules` flag. This tests the cli itself. See other cli tests for more examples.

Reply:
Ahh nice.
I think we can refactor all the test_cli. There is a lot of code duplication there. I will open in issue about it

Reply:
Nice!